share/completions/s3cmd.fish

```diff
@@ -0,0 +1,121 @@
+# s3cmd
+
+function __s3cmd_is_remote_path
+	commandline -ct | string match -r -- "^s3://.*"
```

Comment:
Also, usually you'd use "-q". Completion conditions are silenced, so it's also not necessary here.